spp_farmer_registry_base/models/farm.py

```diff
@@ -111,7 +111,7 @@ def update_group_head_member(self, new_group_head):
     def write(self, vals):
         farm = super().write(vals)
         for rec in self:
-            if rec.is_group:
+            if rec.is_group and rec.kind == self.env.ref("spp_farmer_registry_base.kind_farm").id:
```

Comment:
@emjay0921 This code will raise a UserWarning with a message unsupported operand type since the `rec.kind` and `self.env.ref("spp_farmer_registry_base.kind_farm").id` have different instance.

Please fix it.